electron-starter.js

```diff
@@ -46,21 +46,26 @@ function startMainWindow() {
   mainWindow = new BrowserWindow({
     width: 800,
     height: 600,
+    minWidth: 400,
+    minHeight: 300,
     icon: path.join(__dirname + "/../assets/icons/png/logo.png"),
-    show: false
+    show: !isRunningTests()
```

Comment:
If we wait to show the window until the app is loaded instead of show it directly during the test start process the focus is lost and spectron can't interact with the window properly.